shared/chat/conversation/attachment-fullscreen/index.native.tsx

```diff
@@ -69,11 +69,18 @@ class _Fullscreen extends React.Component<Props & Kb.OverlayParentProps, {loaded
   _setLoaded = () => this.setState({loaded: true})
   render() {
     return (
-      <Kb.SafeAreaViewTop style={styles.safeAreaTop}>
+      <Kb.Box2
+        direction="vertical"
+        style={{backgroundColor: Styles.globalColors.black}}
```

Comment:
this should probably be `globalColors.blackOrWhite` so the background isn't retina-scorchingly bright

Reply:
(on dark mode)

Reply:
👍 this code was from pre-dark mode

Reply:
fixed, also note blackOrWhite means, black in light mode and white in dark mode. we want blackOrBlack

Reply:
ohhhh that's right